toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -48,6 +48,8 @@ func PrintBuildSummary(pkgGraph *pkggraph.PkgGraph, graphMutex *sync.RWMutex, bu
 	builtSRPMs := make(map[string]bool)
```

Comment:
General question for this function - can we log the outputs to a file? It seems like a useful artifact to publish along with the other logs

Reply:
I'll add to my todo list